mocks/react-mocks/src/components/NavBar/index.jsx

```diff
@@ -10,22 +11,22 @@ export default function NavBar() {
         <a href="#mainContent">Skip to Main Content</a>
```

Comment:
Will the skip to main content link need to be updated as well?

Reply:
No, as I mentioned in the above comments to Scott, if it is updated it breaks the skip link.

Reply:
sorry did not see the conversation. If so, we won't need to change the skip the main content for Angular neither right? Cause I am making similar changes for Angular.

Reply:
Also, how did you test accessibility and what does a skip link do?

Reply:
Press tab @Tanya0609  you should see a pop up with skip to main content, press enter if it doesn't take you to main content, something broke.